Core/src/org/sleuthkit/autopsy/coreutils/Logger.java

```diff
@@ -128,13 +96,10 @@ synchronized public static void setLogDirectory(String directoryPath) {
          * add/remove handler calls (currently, the base class handlers
          * collection is a CopyOnWriteArrayList).
          */
-        FileHandler newUserFriendlyHandler = createFileHandlerWithoutTraces(directoryPath);
-        FileHandler newDeveloperFriendlyHandler = createFileHandlerWithTraces(directoryPath);
+        FileHandler newUserFriendlyHandler = createFileHandlerWithTraces(directoryPath);
```

Comment:
Let's thoroughly complete the removal of the concept of user friendly vs. developer friendly logging from this class - it has only been done partially. Please take a few more minutes to make sure that all variable names, etc. in the oced an all comments have been brought up to date to avoid confusing future maintainers of this file.